test/js/data/symbol_bucket.test.js

```diff
@@ -28,7 +28,7 @@ test('SymbolBucket', function(t) {
     }
 
     function bucketSetup() {
-        var bucket = new SymbolBucket(buffers, info, 1);
+        var bucket = new SymbolBucket(buffers, info, 1, 0);
```

Comment:
It's a require field in the Symbol Bucket for determining zoom min and max. If its not passed in it is undefined and leads to NaN zoom levels.